packages/core/src/message-provider.ts

```diff
@@ -91,11 +91,17 @@ export function createMessageProvider({
         });
 
         for (const { _id, encryptedMessage } of encryptedMessages) {
-          await wallet.addDocument({
-            id: _id,
-            type: 'DIDCommMessage',
-            encryptedMessage,
-          });
+          try {
+            await wallet.addDocument({
+              id: _id,
+              type: WalletDocumentTypes.DIDCommMessage,
+              encryptedMessage,
+            });
+          } catch(err) {
+            // this message will be lost if it fails to be stored in the wallet
+            captureException(err);
+            console.error(err);
```

Comment:
there shouldnt be a need to call console.error when you invoke `captureException` - cant `captureException` also do the console log?

Reply:
that's right. I've removed it